include/seastar/core/fair_queue.hh

```diff
@@ -116,7 +119,7 @@ public:
             bi::cache_last<true>,
             bi::member_hook<fair_queue_entry, bi::slist_member_hook<>, &fair_queue_entry::_hook>>;
 
-    fair_queue_ticket ticket() const noexcept { return _ticket; }
+    fair_queue_ticket capacity() const noexcept { return _ticket; }
```

Comment:
capacity is the wrong term here, the disk has capacity (capability to do something), this is something that must fit in capacity. I don't mind if it's renamed post-merge, but we need a more accurate name.

Reply:
It could be "cost", but "cost" is busy with cross-classes balancing: `request.cost = request.capacity / class.shares` and then each class is charged with this "cost", scheduler picks a class with the lowest charge for the next dispatch